textattack/transformations/word_swaps/word_swap_change_location.py

```diff
@@ -43,13 +43,19 @@ def __init__(self, n=3, confidence_score=0.7, **kwargs):
         super().__init__(**kwargs)
         self.n = n
         self.confidence_score = confidence_score
+        self.language = language
 
     def _get_transformations(self, current_text, indices_to_modify):
         words = current_text.words
         location_idx = []
-
+        if self.language == "en":
+            model_name = "ner"
+        elif self.language == "fra" or self.language == "french":
```

Comment:
so here we do not need something like follows? :

`  if self.language == "esp" or self.language == "spanish":`